frontend/src/metabase/query_builder/components/view/ViewHeader/ViewTitleHeader.styled.tsx

```diff
@@ -78,6 +78,26 @@ export const SavedQuestionHeaderButtonContainer = styled.div<{
   right: ${props => (props.isModelOrMetric ? "0px" : "0.38rem")};
 `;
 
+export const FilterButton = styled(MantineButton)`
+  transition: background 300ms linear, border 300ms linear;
+
+  &:hover {
+    color: var(--mb-color-filter);
+    border-color: color-mix(in srgb, var(--mb-color-filter) 30%, white);
+    background-color: color-mix(in srgb, var(--mb-color-filter) 10%, white);
+  }
+` as unknown as typeof MantineButton;
+
+export const SummarizeButton = styled(MantineButton)`
+  transition: background 300ms linear, border 300ms linear;
+
+  &:hover:not([data-active="true"]) {
+    color: var(--mb-color-summarize);
+    border-color: color-mix(in srgb, var(--mb-color-summarize) 40%, white);
+    background-color: color-mix(in srgb, var(--mb-color-summarize) 15%, white);
+  }
+` as unknown as typeof MantineButton;
```

Comment:
I would prefer to do this with CSS modules, but I'm not sure it's possible?

@oisincoveney do let me know if i'm missing something

Reply:
How does this look? #46182 

I don't think there's a barrier to making this one work :smile:

Reply:
inexplicably, I'm seeing the background color being clobbered for the filter button, but not the summarize button 🤯 

![Screen Shot 2024-07-26 at 7 58 56 AM](https://github.com/user-attachments/assets/7bd03311-951c-4c8d-8d64-7a07fbe46911)

![Screen Shot 2024-07-26 at 7 59 06 AM](https://github.com/user-attachments/assets/50234026-b436-4596-bb4b-c3fcd999f15d)

apparently, it's the `:not()` selector that gives it enough specificity to override mantine 🤯 
see: https://github.com/metabase/metabase/pull/46106/commits/666d8cf83180140a58b69df97cb6f62771f9a08b

Reply:
I'm ok with doing this but we should put a comment in there. Though I find it kinda nuts that mantine overrides styles from user-defined classes